app/controllers/OverviewController.scala

```diff
@@ -278,6 +309,12 @@ class OverviewController @Inject()(
               .getOrElse(result)
           }
         case _ =>
+          Future.successful(
+            Redirect(controllers.nonsipp.routes.TaskListController.onPageLoad(srn))
+              .addingToSession(
+                Constants.PREPOPULATION_FLAG -> String.valueOf(lastSubmittedPsrFbInPreviousYears.isDefined)
```

Comment:
Will 'onSelectContinue' got called for submissions(version) greater than 1? As PREPOPULATION_FLAG should not be set for version 1 only.

Reply:
Yes, It will. What should be the value of PREPOPULATION_FLAG for for submissions(version) greater than 1. I thought false made it (`String.valueOf(lastSubmittedPsrFbInPreviousYears.isDefined)`) but I might be wrong

Reply:
as I'm sending None now for previous data (versions greater than 1)

Reply:
"What should be the value of PREPOPULATION_FLAG for for submissions(version) greater than 1?" - I think it should be false.
It's worth double checking, I think.